projects/cups/Dockerfile

```diff
@@ -16,7 +16,7 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 RUN apt-get update && apt-get install -y autoconf libtool-bin pkg-config zlib1g-dev libavahi-client-dev libsystemd-dev
```

Comment:
`autoconf libtool-bin pkg-config` can be removed. They are useless and waste time in the build process.

Reply:
Indeed. I have removed them and tested